ZelBack/config/default.js

```diff
@@ -74,6 +74,7 @@ module.exports = {
     rpcport: 16124,
     porttestnet: 26125,
     rpcporttestnet: 26124,
+    zmqport: 16126,
```

Comment:
This is the default api port.
Will it not cause problems?
Doesn't this needs to also be configured because of upnp nodes?

Reply:
sorry not api, ui port

Reply:
Yup, poor choice there. Have updated.